src/components/Layout.tsx

```diff
@@ -45,9 +46,14 @@ const Layout: FC<PropsWithChildren> = ({ children }) => {
             <div className="flex items-center space-x-10">
               <Link className="group" href={appLink("/")}>
                 <FortressLogo
-                  className="my-6 h-8 w-auto fill-white group-hover:fill-orange-400"
+                  className="my-6 h-8 w-auto fill-white group-hover:hidden"
                   aria-label="Fortress Finance"
                 />
+                <Image
```

Comment:
Just add `priority` as a prop to this image so that we can be (almost) certain it will be loaded by the time the user goes to hover.

Reply:
fixed